### ISSUE TITLE
Handle UTF-8 Characters / Emojis

### ISSUE THREAD

**Issue by geoff-maddock** [geoff-maddock/events-tracker#441]
I noticed when saving some text that contains emojis, I hit a 500 error.
Not sure what the issue is, but if I remove them I can persist.

Might have to do with the database?  Or something else with laravel.

Investigate as it would be good to be able to handle them.

**Comment by geoff-maddock** [comment]
This original event has a number of those characters.  
https://arcane.city/events/2494

From the logs:
```
PDOException: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9D\x90\x81\xF0\x9D...' for column 'description' at row 1
  File "/app/Http/Controllers/EventsController.php", line 1755, in App\Http\Controllers\EventsController::update
    $event->fill($request->input())->save();
  File "/app/Http/Middleware/Authenticate.php", line 44, in App\Http\Middleware\Authenticate::handle
    return $next($request);
  File "/app/Http/Middleware/VerifyCsrfToken.php", line 19, in App\Http\Middleware\VerifyCsrfToken::handle
    return parent::handle($request, $next);
  File "/app/Http/Middleware/VerifyCsrfToken.php", line 19, in App\Http\Middleware\VerifyCsrfToken::handle
    return parent::handle($request, $next);
  File "/public/index.php", line 52
    $request = Illuminate\Http\Request::capture()
...
(54 additional frame(s) were not displayed)
```

**Comment by geoff-maddock** [comment]
Just had to set invidiual text fields to 
Character set: utf8mb4
Collation utf8mb4

**Comment by geoff-maddock** [comment]
Resolved via commit d7d3993a35e4959ca812011bcc5d8cffc15f8e6b
